internal/app/subsystems/aio/store/store.go

```diff
@@ -71,3 +73,13 @@ func Collect(c <-chan *bus.SQE[t_aio.Submission, t_aio.Completion], f <-chan int
 
 	return batch, true
 }
+
+func StoreErr(err error) error {
```

Comment:
Damn, this looks really helpful!

Reply:
I hated that the errors had no context just, `expected 3 args got 4` or something like that. This at least points to the file and line where that happens.